src/reducer.js

```diff
@@ -148,6 +155,7 @@ export default function undoable (reducer, rawConfig = {}) {
     initTypes: parseActions(rawConfig.initTypes, ['@@redux-undo/INIT']),
     limit: rawConfig.limit,
     filter: rawConfig.filter || (() => true),
+    groupBy: rawConfig.groupBy || (() => null),
```

Comment:
Can you please add this option `groupBy` to the `index.d.ts` so typescript users don't see strange errors when defining it.